lib/takeaway.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'menu'
+require_relative 'bill_calculator'
+require_relative 'text'
+class Takeaway
+
+  attr_reader :order, :cost, :get_bill
+
+  def initialize
+    @menu = Menu.new
+    @order = []
+    @get_bill = BillCalculator.new
+  end
+
+  def display_menu
+    @menu.show_menu
+  end
+
+  def select_dish(dish)
```

Comment:
Might be useful to add a 'quantity' variable here so that user's can order more that one helping of a dish at a time.